.github/workflows/ci.yml

```diff
@@ -68,7 +68,7 @@ jobs:
       - name: Use Node.js
         uses: actions/setup-node@v3
         with:
-          node-version: "18.x"
+          node-version: "18.15"
```

Comment:
I had to downgrade due to https://github.com/NomicFoundation/hardhat/issues/3877

Reply:
Nice find